ksql-engine/src/test/java/io/confluent/ksql/services/KafkaTopicClientImplTest.java

```diff
@@ -277,7 +302,7 @@ public void shouldDeleteInternalTopics() {
   }
 
   @Test(expected = TopicDeletionDisabledException.class)
-  public void shouldThrowTopicDeletionDisabledException()
+  public void shouldFailToDeleteThrowTopicDeletionDisabledException()
```

Comment:
nit:
```suggestion
  public void shouldPropagateTopicDeletionDisabledException()
```
or something similar that doesn't make it sound like the test tries to delete an exception.

Same comment for the other tests below as well.